config/prow-staging/cluster/prowjob_customresourcedefinition.yaml

```diff
@@ -12,89 +12,96 @@
 # See the License for the specific language governing permissions and
```

Comment:
@bradmwilliams you also need to change:
* config/prow/cluster/starter-gcs.yaml
* config/prow/cluster/starter-s3.yaml
* prow/test/integration/prow/cluster/100_starter.yaml (most important one as used by integration tests)

Reply:
@alvaroaleman The `starter-*.yaml` files have been accounted for.  Looking at the third file now.

Reply:
Done!